### PR TITLE
redact tooltips

### PR DESCRIPTION
**Pull request by fiji-flo**
Hovering a icon in the commits (either commit tab or conversation tab) currently reveals the author. This should fix it.

### CODE REVIEW

File: github/blind.css [hunk 1]
```diff
@@ -1,3 +1,6 @@
+body.br-blinded .br-label::after {
+  content: attr(redacted-label) !important;
```
Comment (zombie) [inline github/blind.css]: It took me a while to understand this is actually changing the content of the final tooltip element.  Almost too clever. ;)

File: github/common.js [hunk 2]
```diff
@@ -113,6 +113,21 @@ function augment(a) {
   }
 }
 
+function augmentTooltip(a) {
+  const authorElement = document.querySelector("a.author.pull-header-username");
+  const who = a.getAttribute("aria-label");
```
Comment (zombie) [inline github/common.js]: nit: `tooltip` or `label` is probably more descriptive.

File: github/common.js [hunk 3]
```diff
@@ -113,6 +113,21 @@ function augment(a) {
...
+
+  if (!(authorElement && who)) {
+    return;
+  }
+
+  const author = authorElement.textContent.trim();
+  if (who.includes(author)) {
+    a.setAttribute("redacted-label", who.replace(author, "[redacted]"));
```
Comment (zombie) [inline github/common.js]: Why is this needed?  Is there an example tooltip that contains additional text, other than just the author name?

File: github/common.js (hunk 2 above)
Comment (fiji-flo) [reply github/common.js]: Indeed. I just copied this from above.

File: github/common.js (hunk 3 above)
Comment (fiji-flo) [reply github/common.js]: yes this contains `author/REPO:branch` and turns into `[redacted]/REPO:branch`

Comment (fiji-flo) [reply github/common.js]: I guess you're asking why not just set it to `"[redacted]"`? Robustness, maybe to much here. But better safe than sorry :D

Comment (zombie) [reply github/common.js]: Ok, if it's unused, let's drop in the spirit of YAGNI, and keep both js and css simpler.

**Comment by fiji-flo** [comment]
@zombie do you want me to change something else? #33 depends on this and I have one more in the pipeline.

**Comment by zombie** [comment]
Yes, please drop the whole `redacted-label` thing (see [comment above](https://github.com/zombie/blind-reviews/pull/32#discussion_r176858763)) and sorry for the delay.

File: github/common.js (hunk 3 above)
Comment (fiji-flo) [reply github/common.js]: Just to get this right:

Adding the `redacted-label` attribute is needed. I can change this to:
```js
a.setAttribute("redacted-label", "[redacted]");
```

if you think the `replace` is to much here. I can also replace the:
```js
if (who.includes(author)) {
```
with
```js
if (who === author) {
```

I'd argue both of them are clean right now. But I guess that's a matter of preference. You decide.

Comment (zombie) [reply github/common.js]: I don't care much between `===` or `includes`, but the attribute doesn't seem to be needed, so we can just hardcode the `[redacted]` string in css.

Comment (fiji-flo) [reply github/common.js]: I see what you meant. However GitHub made all of this obsolete last week: https://blog.github.com/2018-03-22-user-hovercards/

Thanks anyhow. I'll refactor #33

Comment (zombie) [reply github/common.js]: Ouch, and thanks.
